### PR TITLE
Add mongoose reference ids for models

### PR DESCRIPTION
**Pull request by ChinyangaTL**
Please, confirm that I've made the reference to the right model, and if I have missed any. I also made note of one I wasnt too sure of

### CODE REVIEW

File: models/StallPayment.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
 const StallPaymentSchema = new Schema(
   {
-    vendorId: String,
-    stallId: String,
+    vendorId: {
+      type: mongoose.Types.ObjectId,
+      ref: 'User',
+    },
+    stallId: String, // not sure which this should be
```
Comment (optimusinhisprime) [inline models/StallPayment.js:10]: EventStall

Comment (ChinyangaTL) [reply models/StallPayment.js:10]: Wait, no... I don't see EventStall in the models

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: Its inside the Event file. I wanted to make an array of EventStalls for the stall fileld. I saw something similar in the mongoose docs

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: Like how they did it here

https://mongoosejs.com/docs/schematypes.html#arrays

Comment (ChinyangaTL) [reply models/StallPayment.js:10]: Slightly lost... If it isn't a mongoose object with a schema, i think we can merge as is.
But make the change if you know exactly what's what.

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: Oh yea. I was saying that the EventStall schema is inside the event file. I wanted to do what the link shows. 

As for the Id it just becomes {
      type: mongoose.Types.ObjectId,
      ref: 'EventStall',
    },
    
   Unless the ref is for the file?

Comment (ChinyangaTL) [reply models/StallPayment.js:10]: May i have the file path to the events file.

I only see events.js

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: Mara it doesnt look like it. Can i please get a link for more info on refs?

Comment (ChinyangaTL) [reply models/StallPayment.js:10]: But your logic would be sound if it is a proper schema

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: This one? http://localhost:3000/vendor/events/1

Comment (optimusinhisprime) [reply models/StallPayment.js:10]: vendor/ 

displays the index which shows all events

**Comment by optimusinhisprime** [comment]
@ChinyangaTL Are you cool with this?

**Comment by ChinyangaTL** [comment]
@optimusinhisprime, check conflict

**Comment by optimusinhisprime** [comment]
On it.

**Pull request merged**